src/wechaty/user/room.py

```diff
@@ -140,7 +140,7 @@ async def find_all(cls, room_id: Optional[str] = None,
                 if room_id == room.room_id:
                     room_result.append(room)
 
-                elif topic is not None and room.topic() == topic:
+                elif topic is not None and await room.topic() == topic:
```

Comment:
`await` gramma should not be in control style code.  My suggestion is :
```python
elif topic is not None:
    room_topic = await room.topic()
    if room_topic == topic:
        # do something
```

Reply:
I think this is python's essence, but never mind, new pr is coming.